flang/runtime/command.cpp

```diff
@@ -15,6 +15,14 @@
 #include <cstdlib>
 #include <limits>
 
+#ifdef _WIN32
+// On Windows GetCurrentProcessId returns a DWORD aka uint32_t
```

Comment:
You're missing:
```
#define WIN32_LEAN_AND_MEAN
#define NOMINMAX
#include <windows.h>
```
just above `#include <processthreadsapi.h>`. 
You can't include processthreadsapi.h without windows.h having been included at some point previously

Reply:
Thanks for the comment, file has been changed accordingly